Code/GraphMol/SmilesParse/CXSmilesOps.cpp

```diff
@@ -2490,7 +2500,7 @@ std::string getCXExtensions(const ROMol &mol, std::uint32_t flags) {
   }
 
   const Conformer *conf = nullptr;
-  if (mol.getNumConformers()) {
+  if (mol.getNumConformers() && (flags & SmilesWrite::CX_COORDS)) {
```

Comment:
I think this change means that the code starting at line 2523 below can be simplified to:
```
  else if (flags & SmilesWrite::CXSmilesFields::CX_BOND_ATROPISOMER) {
    Atropisomers::wedgeBondsFromAtropisomers(mol, conf, wedgeBonds);
    const auto cfgblock = get_bond_config_block(
        mol, atomOrder, bondOrder, conf != nullptr, wedgeBonds, true);
    appendToCXExtension(cfgblock, res);
  }
```

Reply:
that works